src/containers/Validators/VotingTab.tsx

```diff
@@ -0,0 +1,124 @@
+import { useTranslation } from 'react-i18next'
+import { FC } from 'react'
+import axios from 'axios'
+import { useQuery } from 'react-query'
+
+import { ValidatorSupplemented } from '../shared/vhsTypes'
+import { SimpleRow } from '../shared/components/Transaction/SimpleRow'
+
+import './votingTab.scss'
+import { renderXRP } from '../Ledgers/LedgerMetrics'
+import {
+  FETCH_INTERVAL_ERROR_MILLIS,
+  FETCH_INTERVAL_VHS_MILLIS,
+  SERVER_ERROR,
+} from '../shared/utils'
+import { useAnalytics } from '../shared/analytics'
+import { RouteLink } from '../shared/routing'
+import { AMENDMENT_ROUTE } from '../App/routes'
+
+const DROPS_TO_XRP_FACTOR = 1000000
+
+export const VotingTab: FC<{
+  validatorData: ValidatorSupplemented
+  network: string | undefined
+}> = ({ validatorData, network }) => {
+  const { t } = useTranslation()
+  const { trackException } = useAnalytics()
+
+  const votedAmenments = new Set(
+    validatorData.amendments
+      ? validatorData.amendments.map((amendment) => amendment.id)
+      : [],
+  )
+  const { data } = useQuery<Array<{ id: string; name: string }>>(
+    ['fetchNetworkVotingData', network],
+    async () => fetchNetworkVote(network),
+    {
+      refetchInterval: (returnedData, _) =>
+        returnedData == null
+          ? FETCH_INTERVAL_ERROR_MILLIS
+          : FETCH_INTERVAL_VHS_MILLIS,
+      refetchOnMount: true,
+      enabled: !!network,
+    },
+  )
+
+  function fetchNetworkVote(networkID: string | undefined) {
+    const url = `${process.env.VITE_DATA_URL}/amendments/vote/${networkID}`
+    return axios
+      .get(url)
+      .then((resp) => resp.data)
+      .then((response) =>
+        response.amendments.voting.amendments.map((amendment) => ({
+          id: amendment.id,
+          name: amendment.name,
+        })),
+      )
```

Comment:
Why does this need to be mapped? Why not just this?
```suggestion
      .then((response) => response.amendments.voting.amendments),
      )
```